budgets/repo.go

```diff
@@ -12,7 +14,8 @@ func NewBudgetRepo(db *database.Database) *BudgetRepo {
 
 func (b *BudgetRepo) Get(id int) (*database.Budget, error) {
 	var budget database.Budget
-	err := b.db.Preload("BudgetCategories.Category").Find(&budget, 1).Error
+
+	err := b.db.Preload("BudgetCategories.Category").Preload("BudgetCategories.Payments").Find(&budget, 1).Error
```

Comment:
Getting long here, consider:

```
err := b.db.Preload("BudgetCategories.Category").
    Preload("BudgetCategories.Payments").
    Find(&budget, 1).
    Error
```

For readability. More lines, yes, but each line (roughly) does one thing.